devlib/target.py

```diff
@@ -62,10 +62,11 @@ def is_connected(self):
         return self.conn is not None
 
     @property
-    @memoized
```

Comment:
I guess this has to go if we want to switch the property values using the adb_root(enable) call.
Unless there is some API I'm not aware of to update the cached values of a memoized...

Moreover, the setattr cannot be used in this property because the way we set it depends on Linux/AndroidTarget implementation... unless I can defined different connected_as_root.setattr for each derived class. :-?